assets/app/view/welcome.rb

```diff
@@ -23,33 +22,34 @@ def render_notification
       message = <<~MESSAGE
         <p>1846 is now available for early alpha. It would help most if you played with the intent to find bugs in hot seat mode.
         The private companies have not been implemented yet.
-        <p/>
+        </p>
 
-        Please file issues <a href='https://github.com/tobymao/18xx/issues'>here</a>. And if you have any questions, check out the
+        <p>Please file issues <a href='https://github.com/tobymao/18xx/issues'>here</a>. And if you have any questions, check out the
         <a href='https://docs.google.com/document/d/1nCYnzNMQLrFLZtWdbjfuSx5aIcaOyi27lMYkJxcGayw/edit'>FAQ!</a>
+        </p>
 
         <p>If you're looking to buy these games, please check out
         <a href='https://all-aboardgames.com'>All-Aboard Games</a>,
         <a href='https://www.grandtrunkgames.com'>Grand Trunk Games</a>,
         and <a href='https://www.gmtgames.com/'>GMT Games</a>.
         </p>
 
-        You can support this project on <a href='https://www.patreon.com/18xxgames'>Patreon</a>.
+        <p>You can support this project on <a href='https://www.patreon.com/18xxgames'>Patreon</a>.
         </p>
       MESSAGE
 
       props = {
         style: {
           background: 'rgb(240, 229, 140)',
           color: 'black',
-          'padding': '1em',
+          marginBottom: '1rem',
```

Comment:
should it be margin-bottom?

Reply:
Either `'margin-bottom'` or `marginBottom`, cf. https://github.com/snabbdom/snabbdom/blob/d9c4abd7c5b166ff7abd705fc920eb0933b9f0a0/README.md#example
CamelCase is just easier to type.

Reply:
didn't realize you could do that